### PR TITLE
feat: Add four Agile questions

### PR DESCRIPTION
**Pull request by danieled-it**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

As per commit message, just added four questions related to Agile, relates to #103

### CODE REVIEW

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/full-quiz.ts [hunk 1]
```diff
@@ -7196,5 +7196,49 @@ const fullQuiz = [
...
   },
+  {
+    Question:
+      "In a Scrum team, what is a Scrum Master?",
+    Answer: "A servant leader whose role is to support the rest of the team and the organization",
+    Distractor1: "A project manager who's main role is to ensure the Developers deliver in time",
+    Distractor2: "An external resource temporarily assigned to the team to audit their work",
+    Distractor3: "Scrum Master is not a role officially recognized by the Scrum guide",
+    Explanation:
+      "The Scrum Master is a leader who supports their colleagues in their activities and does more than just assign tasks."
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:7207]: The automated check for the linter failed because we are missing a comma at the end

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: src/data/full-quiz.ts (hunk 1 above)
Comment (danieled-it) [reply src/data/full-quiz.ts]: I know, fixed and committed.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first time contribution to the Developer Quiz Site. 🎉
We look forward to future contributions. 👍

**Pull request merged**